### DISCUSSION TITLE
What is stmt? any examples to show how to use?

### DISCUSSION THREAD

**Discussion by kolinfluence** [crossdb-org/crossdb#28] [Q&A]
https://crossdb.org/client/api-c/?h=stmt

not sure how to use

**Comment by jcwangxp** [comment]
https://crossdb.org/develop/prep-stmts/

**Reply by jcwangxp** [reply]
You can check the SQL and Prepared statement(STMT) performance for both sqlite3 and CrossDB.
https://crossdb.org/blog/benchmark/crossdb-vs-sqlite3/
